concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -268,17 +268,17 @@ jobs:
             - -e
             - -c
             - |
-              paas-bootstrap/concourse/scripts/s3init.sh {{state_bucket}} bosh.tfstate paas-bootstrap/concourse/init_files/terraform.tfstate.tpl
-              paas-bootstrap/concourse/scripts/s3init.sh {{state_bucket}} {{bosh_manifest_state}} paas-bootstrap/concourse/init_files/bosh-init-state.json.tpl
-              paas-bootstrap/concourse/scripts/s3init.sh {{state_bucket}} vpc.tfstate paas-bootstrap/concourse/init_files/terraform.tfstate.tpl
-              paas-bootstrap/concourse/scripts/s3init.sh {{state_bucket}} bosh-CA.tar.gz paas-bootstrap/concourse/init_files/empty.tar.gz
-              paas-bootstrap/concourse/scripts/s3init.sh {{state_bucket}} bosh-secrets.yml paas-bootstrap/concourse/init_files/zero_bytes
-              paas-bootstrap/concourse/scripts/s3init.sh {{state_bucket}} bosh_id_rsa paas-bootstrap/concourse/init_files/zero_bytes
-              paas-bootstrap/concourse/scripts/s3init.sh {{state_bucket}} bosh_id_rsa.pub paas-bootstrap/concourse/init_files/zero_bytes
-              paas-bootstrap/concourse/scripts/s3init.sh {{state_bucket}} id_rsa paas-bootstrap/concourse/init_files/zero_bytes
-              paas-bootstrap/concourse/scripts/s3init.sh {{state_bucket}} id_rsa.pub paas-bootstrap/concourse/init_files/zero_bytes
-              paas-bootstrap/concourse/scripts/s3init.sh {{state_bucket}} concourse.tfstate paas-bootstrap/concourse/init_files/terraform.tfstate.tpl
-              paas-bootstrap/concourse/scripts/s3init.sh {{state_bucket}} concourse-secrets.yml paas-bootstrap/concourse/init_files/zero_bytes
+              paas-bootstrap/concourse/scripts/s3init.sh "((state_bucket))" bosh.tfstate paas-bootstrap/concourse/init_files/terraform.tfstate.tpl
+              paas-bootstrap/concourse/scripts/s3init.sh "((state_bucket))" ((bosh_manifest_state)) paas-bootstrap/concourse/init_files/bosh-init-state.json.tpl
```

Comment:
```"((bosh_manifest_state))"```

Reply:
you're to good at it... changed, thanks.